### DISCUSSION TITLE
Set output file name

### DISCUSSION THREAD

**Discussion by Sen-Nyin** [glenn2223/vscode-live-sass-compiler#183] [Q&A] [answered]
Hi there, is there currently (or planned) any way to set Live Sass Compiler to output a specific  .css file name? For example, if I am working in 'main.scss' the compiled file would be style.css.

**Comment by glenn2223** [comment, marked as answer]
Unfortunately, this isn't planned. As the extension can compile many CSS files (rather than the one in your use case), there's no way of changing the name without risking users overwriting files.

However, if this feature is heavily requested and someone can think of a sensible way of implementing it then I'm open to it.
